share/spice/metar/metar.js

```diff
@@ -0,0 +1,29 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_metar = function(api_result) {
+        
+        if (!api_result || api_result.Error) {
+            return Spice.failed('metar');
+        }    
+
+        Spice.add({
+            id: "metar",
+            name: 'Weather',
+            data: api_result,
+            meta: {
+                sourceName: "Aviation Weather REST API",
+                sourceUrl: "http://avwx.rest"
+            },
+            normalize: function(item) {
+                return {
+                    title: item["Station"], 
+                    subtitle: item["Time"],
```

Comment:
> One question, are strings internationalized, or I can just display the text in English?

Great question. Right now they are not internationalized. Nearly all our Instant Answers use English APIs and show English results.